mne/data/coil_def.dat

```diff
@@ -510,72 +468,127 @@
  -0.1250  4.082e-03 -7.071e-03  5.000e-02  0.000  0.000  1.000
  -0.1250 -4.082e-03  7.071e-03  5.000e-02  0.000  0.000  1.000
  -0.1250 -4.082e-03 -7.071e-03  5.000e-02  0.000  0.000  1.000
-2   7501   0    2  1.486e-02  5.740e-02 "Artemis123 axial gradiometer size = 1.486e-02 mm base = 5.740e-02 mm"
+0   7501    0   2  1.486e-02  5.740e-02	"Artemis system gradiometer size = 14.86  mm base = 57.40  mm"
   1.0000  0.000e+00  0.000e+00  0.000e+00  0.000  0.000  1.000
  -1.0000  0.000e+00  0.000e+00  5.740e-02  0.000  0.000  1.000
-2   7501   1    8  1.486e-02  5.740e-02 "Artemis123 axial gradiometer size = 1.486e-02 mm base = 5.740e-02 mm"
+0   7501    1   8  1.486e-02  5.740e-02	"Artemis system gradiometer size = 14.86  mm base = 57.40  mm"
   0.2500  3.715e-03  3.715e-03  0.000e+00  0.000  0.000  1.000
-  0.2500  3.715e-03 -3.715e-03  0.000e+00  0.000  0.000  1.000
   0.2500 -3.715e-03  3.715e-03  0.000e+00  0.000  0.000  1.000
   0.2500 -3.715e-03 -3.715e-03  0.000e+00  0.000  0.000  1.000
+  0.2500  3.715e-03 -3.715e-03  0.000e+00  0.000  0.000  1.000
  -0.2500  3.715e-03  3.715e-03  5.740e-02  0.000  0.000  1.000
- -0.2500  3.715e-03 -3.715e-03  5.740e-02  0.000  0.000  1.000
  -0.2500 -3.715e-03  3.715e-03  5.740e-02  0.000  0.000  1.000
  -0.2500 -3.715e-03 -3.715e-03  5.740e-02  0.000  0.000  1.000
-2   7501   2   14  1.486e-02  5.740e-02 "Artemis123 axial gradiometer size = 1.486e-02 mm base = 5.740e-02 mm"
+ -0.2500  3.715e-03 -3.715e-03  5.740e-02  0.000  0.000  1.000
+0   7501    2  14  1.486e-02  5.740e-02	"Artemis system gradiometer size = 14.86  mm base = 57.40  mm"
   0.2500  0.000e+00  0.000e+00  0.000e+00  0.000  0.000  1.000
-  0.1250  6.066e-03  0.000e+00  0.000e+00  0.000  0.000  1.000
-  0.1250 -6.066e-03  0.000e+00  0.000e+00  0.000  0.000  1.000
-  0.1250  3.033e-03  5.253e-03  0.000e+00  0.000  0.000  1.000
-  0.1250  3.033e-03 -5.253e-03  0.000e+00  0.000  0.000  1.000
-  0.1250 -3.033e-03  5.253e-03  0.000e+00  0.000  0.000  1.000
-  0.1250 -3.033e-03 -5.253e-03  0.000e+00  0.000  0.000  1.000
+  0.1250  6.067e-03  0.000e+00  0.000e+00  0.000  0.000  1.000
+  0.1250 -6.067e-03  0.000e+00  0.000e+00  0.000  0.000  1.000
+  0.1250  3.033e-03  5.254e-03  0.000e+00  0.000  0.000  1.000
+  0.1250  3.033e-03 -5.254e-03  0.000e+00  0.000  0.000  1.000
+  0.1250 -3.033e-03  5.254e-03  0.000e+00  0.000  0.000  1.000
+  0.1250 -3.033e-03 -5.254e-03  0.000e+00  0.000  0.000  1.000
  -0.2500  0.000e+00  0.000e+00  5.740e-02  0.000  0.000  1.000
- -0.1250  6.066e-03  0.000e+00  5.740e-02  0.000  0.000  1.000
- -0.1250 -6.066e-03  0.000e+00  5.740e-02  0.000  0.000  1.000
- -0.1250  3.033e-03  5.253e-03  5.740e-02  0.000  0.000  1.000
- -0.1250  3.033e-03 -5.253e-03  5.740e-02  0.000  0.000  1.000
- -0.1250 -3.033e-03  5.253e-03  5.740e-02  0.000  0.000  1.000
- -0.1250 -3.033e-03 -5.253e-03  5.740e-02  0.000  0.000  1.000
-1   7502   0    1  1.485e-02  0.000e+00 "Artemis123 reference magnetometer size = 1.485e-02 mm"
```

Comment:
Same here, used to be type 1 (mag) and now is type 0

Reply:
Will investigate on this.